common/flatpak-transaction.c

```diff
@@ -1889,6 +1891,84 @@ flatpak_transaction_get_include_unused_uninstall_ops (FlatpakTransaction *self)
   return priv->include_unused_uninstall_ops;
 }
 
+/**
+ * flatpak_transaction_set_auto_install_sdk:
+ * @self: a #FlatpakTransaction
+ * @auto_install_sdk: whether to auto install SDKs for apps
+ *
+ * When this is set to %TRUE, Flatpak will automatically install the SDK for
+ * each app currently being installed or updated. Does nothing if an uninstall
+ * is taking place.
+ *
+ * Since: 1.9.4
+ */
+void
+flatpak_transaction_set_auto_install_sdk (FlatpakTransaction *self,
+                                          gboolean            auto_install_sdk)
+{
+  FlatpakTransactionPrivate *priv = flatpak_transaction_get_instance_private (self);
+
+  priv->auto_install_sdk = auto_install_sdk;
+}
+
+/**
+ * flatpak_transaction_get_auto_install_sdk:
+ * @self: a #FlatpakTransaction
+ *
+ * Gets the value set by
+ * flatpak_transaction_set_auto_install_sdk().
+ *
+ * Returns: %TRUE if auto_install_sdk is set, %FALSE otherwise
+ *
+ * Since: 1.9.4
+ */
+gboolean
+flatpak_transaction_get_auto_install_sdk (FlatpakTransaction *self)
+{
+  FlatpakTransactionPrivate *priv = flatpak_transaction_get_instance_private (self);
+
+  return priv->auto_install_sdk;
+}
+
+/**
+ * flatpak_transaction_set_auto_install_debug:
+ * @self: a #FlatpakTransaction
+ * @auto_install_debug: whether to auto install debug info for apps
+ *
+ * When this is set to %TRUE, Flatpak will automatically install the debug
+ * info for each app currently being installed or updated. Does nothing if an
+ * uninstall is taking place.
+ *
+ * Since: 1.9.4
```

Comment:
```suggestion
 * Since: 1.13.3
```

Reply:
This doesn't seem resolved despite being marked resolved

Reply:
This is because I can't git.